public/app/ui/pages/about/AboutPage.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    'use strict';
+
+    define(
+        [
+            'ui/responsive'
+        ],
+        function (responsive) {
+            return function () {
+                this.ready = function (element, callback) {
```

Comment:
This `ready` callback is executed once the view model is initialised and bound to the template, and after the `binding` event has completed.  See `app/util/bind.js`.